pages/sign-in.js

```diff
@@ -1,19 +1,12 @@
 import React from 'react';
 import Image from 'next/image';
 import Link from 'next/link';
-import * as yup from 'yup';
 import { Button, Input, Label, Navbar } from '@/components/index';
 import HookForm from '@/components/form/Form';
 import { LogoBlack, SignInImage } from '/public/assets/signUp';
+import { SignInSchema } from '../schema/Schema';
```

Comment:
This should be in `authSchema`.

```suggestion
import { SignInSchema } from '../schema/authSchema';
```